### DISCUSSION TITLE
Does 100k label file contain occlusion and truncation values?

### DISCUSSION THREAD

**Discussion by sainisanjay** [bdd100k/bdd100k#46] [Q&A]
I can see in the `"det_v2_train_release.json"` only has` "attributes": {"occluded": false, "truncated": false,`.
Is there any way to get occluded and truncation values?

**Comment by XiaLiPKU** [comment]
Do you mean "their values are always false"? I re-check the json file and find that is not the case, you can still find "true" for these attributes.

**Reply by sainisanjay** [reply]
No, their values are either `true `or `false`. When attribute is` true` then how I will know, how much object is occluded?
